app/assets/javascripts/foreman_virt_who_configure/config_copy_and_paste.js

```diff
@@ -0,0 +1,8 @@
+function virt_who_copy_configuration_to_clipboard() {
+  var aux = document.createElement("textarea");
+  $(aux).html($('pre.terminal').html());
```

Comment:
Nitpick: `$('pre.terminal').html()` could be passed as a param to decouple the function from the html code.